src/mgr/MgrStandby.cc

```diff
@@ -205,7 +206,13 @@ void MgrStandby::handle_signal(int signum)
   Mutex::Locker l(lock);
   assert(signum == SIGINT || signum == SIGTERM);
   derr << "*** Got signal " << sig_str(signum) << " ***" << dendl;
-  shutdown();
+
+  // FIXME: we can't reliably shut down the python environment.  Just exit.
```

Comment:
Please use "can not" instead of can't, as per coding guidelines.
An PR is raised and accepted for the same.

Reply:
plz ignore